docs/330-contributing.md

```diff
@@ -0,0 +1,37 @@
+Welcome to OVOS! First off, thanks for taking the time to contribute! ❤️
+
+If you're not familiar with us, [our GoFundMe page has a brief history of what we do and how we got here](https://www.gofundme.com/f/openvoiceos). (Please don't feel obligated to contribute to the GoFundMe unless you really want to though - we're okay right now and you're here to help code!)
+
+We're a very friendly and open community and most responsive on [Matrix](https://matrix.to/#/#OpenVoiceOS-Support:matrix.org). Please come say hi and feel free to ask any questions.
+
+## How can you help?
+
+In short, OVOS has found ourselves in the position of having to take a large amount of R&D code and polish it up for a broader user base. We welcome PRs at the best of times, but we especially need help with things that may not be marked as issues:
+
+1. Adding docstrings to methods, especially on skills (starts with `skill-ovos-`) and plugins (has `plugin` in the name somewhere).
+2. Localization. We strive to be an international private and open source voice assistant, and it's a huge contribution to translate our skills to your language! No, really! #goodfirstissue
+3. Writing tests. Nobody loves it, everybody needs it, and we love if you do it!
+4. Documentation. Both our [technical manual](https://openvoiceos.github.io/ovos-technical-manual/) and our [community docs](https://openvoiceos.github.io/community-docs/) are community-managed and could use PRs ranging from grammatical fixes, fixing broken links, and clarifying/expanding on existing information. #goodfirstissue
+5. Testing out skills and looking for flaws/ways to improve. Many of our skills have been released and in the wild for years, but could use some loving attention. You can find actively supported skills in the `OpenVoiceOS` org, or skills in need of some care in the `OVOSHatchery` org.
+
+## How do you get started?
+
+First of all, if you're just starting out and want to contribute documentation or localization PRs, don't sweat it if you don't have OVOS running. We're happy to help with testing those. If you'd like to run it yourself and confirm things work as you expect, read on.
+
+OVOS has a [reference setup for Docker](https://github.com/OpenVoiceOS/ovos-docker) that will run on any platform and architecture that uses Docker - Linux, Mac OS, and Windows with WSL2. If you're comfortable with Docker, this is the fastest and easiest way to get OVOS running locally to contribute back. (Yes, it also works with Podman!)
```

Comment:
up to you guys, but do we still need this sentence? 
> this is the fastest and easiest way to get OVOS running locally to contribute back.

since we have ovos-installer mentioned below it should cover this, installer also supports WSL2  (but not Mac yet)

i'd argue a venv install is better to get a dev setup instead of docker if the goal is to contribute back

Reply:
I agree with this also

Reply:
I also think the best way is to use installer.